addons/website_event_track/views/event_track_templates_page.xml

```diff
@@ -3,28 +3,30 @@
 
 <template id="event_track_main" name="Event Exhibitor">
     <t t-call="website_event.layout">
-        <t t-set="navbar__back_url" t-value="'/event/%s/track' % (slug(event))"/>
-        <t t-set="navbar__back_title">Back to All Talks</t>
-        <t t-set="navbar__back_text">All Talks</t>
+        <!-- Set aside block only if there are filtered tracks -->
+        <t t-set="has_ongoing_or_upcoming" t-value="any(track.is_track_live or track.track_start_remaining for track in tracks_other)"/>
+        <t t-set="filtered_tracks_other" t-value="tracks_other.filtered(lambda track: not (has_ongoing_or_upcoming and track.is_track_done))"/>
 
-        <div class="o_wevent_online o_wesession_index">
-            <!-- Options -->
-            <t t-set="option_widescreen" t-value="option_widescreen or False"/>
-            <t t-set="option_track_wishlist" t-value="not event.is_done and is_view_active('website_event_track.session_topbar_wishlist')"/>
-            <!-- Drag/Drop Area -->
-            <div id="oe_structure_wesession_track_index_1" class="oe_structure"/>
-            <!-- Content -->
```

Comment:
Why do we remove that ?

I think we still need the drag-and-drop area (i.e: the element with the `oe_structure` class). It allows the user to add an element there when using the website builder. The variables also seem to be necessary for the bell icon. I think we have tests failing because of that :sweat_smile:

Reply:
Ops Thanks for point out :+1: